packages/client/src/Call.ts

```diff
@@ -1929,10 +1968,13 @@ export class Call {
       )
       .subscribe((p) => {
         const source = kind === 'video' ? p.videoStream : p.screenShareStream;
-        if (videoElement.srcObject === source) return;
+        if (videoElement.srcObject === source || !source) return;
 
         setTimeout(() => {
-          videoElement.srcObject = source ?? null;
+          videoElement.srcObject = source;
+          videoElement.play().catch((e) => {
+            this.logger('warn', `Failed to play stream`, e);
+          });
```

Comment:
Wouldn't be this covered by the `autoplay` attribute or does resetting the source cause the autoplay to not work?

Reply:
Technically it should, but at some point, there were some issues with it in Firefox and Safari, and explicit invocation of `play()` helped to resolve them. I'd like to keep the same code